src/atomate2/lobster/schemas.py

```diff
@@ -209,6 +211,9 @@ class LobsterTaskDocument(BaseModel):
     COBI_data: CompleteCohp = Field(
         None, description="pymatgen CompleteCohp object with COBI data"
     )
+    lobster_dos: LobsterCompleteDos = Field(
+        None, description="pymatgen pymatgen.io.lobster.Doscar.completedos data"
+    )
     lobster_charges: Any = Field(
         None,
         description="Atomic charges dict from LOBSTER based on Mulliken and L�wdin",
```

Comment:
ö might need to be replaced. Probably my fault.

Reply:
No,no its not your fault. I did it. My mistake. Will fix it. 

Also, I think storing the Madelung energies from Mulliken and Löwdin would be better seperately rather than just including only Madelung energies based on Mulliken charges in lobsterpy output.

Reply:
Very true.